tests/pds/dataservice_test.go

```diff
@@ -396,6 +396,111 @@ var _ = Describe("{ScaleUPDataServices}", func() {
+				pdsNamespaceLabel: "true",
+			}
+			testns, err := pdslib.UpdatePDSNamespce(ns, nsLables)
+			log.FailOnError(err, "Error while updating pds namespace")
+			log.InfoD("PDS Namespace Updated with PDS Label %v", testns)
+		})
+		Step("Deploy, Validate and Delete Data Services", func() {
+			for _, ds := range params.DataServiceToTest {
+				if ds.Name == postgresql {
+					log.InfoD("Deploying DataService %v ", ds.Name)
+					isDeploymentsDeleted = false
+					dataServiceDefaultResourceTemplateID, err = pdslib.GetResourceTemplate(tenantID, ds.Name)
+					log.FailOnError(err, "Error while getting resource template")
+					log.InfoD("dataServiceDefaultResourceTemplateID %v ", dataServiceDefaultResourceTemplateID)
+
+					dataServiceDefaultAppConfigID, err = pdslib.GetAppConfTemplate(tenantID, ds.Name)
+					log.FailOnError(err, "Error while getting app configuration template")
+					dash.VerifyFatal(dataServiceDefaultAppConfigID != "", true, "Validating dataServiceDefaultAppConfigID")
+					log.InfoD(" dataServiceDefaultAppConfigID %v ", dataServiceDefaultAppConfigID)
+					namespaceID, err := pdslib.GetnameSpaceID(ns, deploymentTargetID)
+					log.FailOnError(err, "error while getting namespaceid")
+					deployment, _, dataServiceVersionBuildMap, err = pdslib.DeployDataServices(ds.Name, projectID,
+						deploymentTargetID,
+						dnsZone,
+						deploymentName,
+						namespaceID,
+						dataServiceDefaultAppConfigID,
+						int32(ds.Replicas),
+						serviceType,
+						dataServiceDefaultResourceTemplateID,
+						storageTemplateID,
+						ds.Version,
+						ds.Image,
+						ns,
+					)
+					log.FailOnError(err, "Error while deploying data services")
+
+					Step("Validate Storage Configurations", func() {
+						resourceTemp, storageOp, config, err := pdslib.ValidateDataServiceVolumes(deployment, ds.Name, dataServiceDefaultResourceTemplateID, storageTemplateID, ns)
+						log.FailOnError(err, "error on ValidateDataServiceVolumes method")
+						ValidateDeployments(resourceTemp, storageOp, config, ds.Replicas, dataServiceVersionBuildMap)
+					})
+
+					Step("Delete Deployments", func() {
+						log.InfoD("Deleting DataService %v ", ds.Name)
+						resp, err := pdslib.DeleteDeployment(deployment.GetId())
+						log.FailOnError(err, "Error while deleting data services")
+						dash.VerifyFatal(resp.StatusCode, http.StatusAccepted, "validating the status response")
+						isDeploymentsDeleted = true
+					})
+				}
+			}
+		})
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+
+		defer func() {
+			log.InfoD("Trying to Delete Independent App pod now : %s", podName)
+			pdslib.DeleteK8sPods(podName, ns)
+			log.InfoD("Trying to Delete Independent PVC now from ns : %s", ns)
+			k8sCore.DeletePersistentVolumeClaim("mysql-pvc", ns)
+			log.InfoD("Trying to delete Independent PV now")
+			k8sCore.DeletePersistentVolume("mysql-pv")
+			log.InfoD("Trying to delete NS now: %s", ns)
+			pdslib.DeletePDSNamespace(ns)
```

Comment:
are there no returns for DeleteK8sPods, DeletePersistentVolumeClaim,   DeletePersistentVolume and DeletePDSNamespace